iam_migrations/freeipa/Cargo.toml

```diff
@@ -34,6 +33,9 @@ uuid = { workspace = true, features = ["serde"] }
 # For file metadata, should this me moved out?
 kanidmd_lib.workspace = true
 
+[target.'cfg(not(target_family = "windows"))'.dependencies]
```

Comment:
Probably makes more sense for this to be "target_family = unix" since the options are unix, windows, wasm, and neither windows or wasm support this.

Reply:
True, Done!